browser/js/project/project.state.js

```diff
@@ -14,7 +14,7 @@ app.config(function ($stateProvider) {
     });
 });
 
-app.controller('ProjectController', function ($scope, AllProjects, ProjectFactory, $state) {
+app.controller('ProjectController', function ($scope, AllProjects, ProjectFactory, $state, $sce) {
```

Comment:
That's for angular to make sure that the url we provided is a safe place
I forget to delete it lol
but maybe we will use it when we deploy.   I am not sure.
for now the ng-src seems working.  not sure if its going to work when we deploy